rtd_docs/conf.py

```diff
@@ -49,4 +49,18 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
\ No newline at end of file
+html_static_path = ['_static']
+
+# -- Run Doxygen for C++ API --------------------------------------------------
+
+read_the_docs_build = os.environ.get('READTHEDOCS', None) == 'True'
+
+if read_the_docs_build:
+    subprocess.call('doxygen Doxyfile', shell=True)
+
+# -- Breathe Configuration ----------------------------------------------------
+
+breathe_projects = { "ISIS": "doxyxml/" }
+breathe_default_project = "ISIS"
+
+breathe_domain_by_extension = {"h" : "cpp"}
```

Comment:
Is this related to the core stuff? Seems like an odd inclusion. Im guessing it's for building the PVL core with docs. If so we might want to add a new objective of getting the docs pushed to the S3 buckets and the landing page, but that can happen after this is merged.